tests/test_clientsdaily.py

```diff
@@ -0,0 +1,69 @@
+import datetime as DT
+import pytest
+import json
+import os
+from mozetl.schemas import MAIN_SUMMARY_SCHEMA
+
+EXPECTED_INTEGER_VALUES = {
+    'active_addons_count_mean': 3613,
+    'crashes_detected_content_sum': 9,
+    'first_paint_mean': 12802105,
+    'pings_aggregated_by_this_row': 1122,
+    'search_count_sum': 1043
+}
+
+
+@pytest.fixture
+def make_frame(spark):
+    root = os.path.dirname(__file__)
+    path = os.path.join(root, 'resources',
+                        'main_summary-late-may-1123-rows-anonymized.json')
+    with open(path) as f:
+        jrows = json.load(f)
+        jrdd = spark.sparkContext.parallelize(map(json.dumps, jrows))
+        frame = spark.read.json(jrdd, MAIN_SUMMARY_SCHEMA)
+    return frame
+
+
+@pytest.fixture
+def make_frame_with_extracts(spark):
+    from mozetl.clientsdaily import rollup as R
```

Comment:
Imports should be module level. I would also avoid the 1 letter aliasing of namespaces (`pyspark.sql.functions as F` is contradictory to this statement, but it's also used as part of a domain specific language that has a fair number of conflicts with python built-ins).